content/02.author-sections.md

```diff
@@ -330,6 +330,8 @@ text = parsed["content"]
 
 
 <!-- Shantanu Jain -->
+### Shantanu Jain
+Hi all, I am very excited to be here attending PSB. I am a reasearch scientist at Northeastern University. I am broadly interested in Machine learning methods. During my Ph.D., I worked on positive unlabled learning. I am most proud aboput my research on non parametric estimation of class priors from positvie and unlabeld data [@arXiv:1601.01944]. I have started learning about Causal Inference lately and I am interested in applying it to Biological datasets.
```

Comment:
Please add a newline after each sentence, for instance:
```
Hi all, I am very excited to be here attending PSB.
I am a reasearch scientist at Northeastern University.
I am...
```

A few typos:
- reasearch 
- unlabled 
- aboput 
- positvie 
- unlabeld 

The citations types are case sensitive.  Please try `[@arxiv:1601.01944]`.

Finally, please add yourself to the author list by editing `metadata.yaml`.